code/modules/surgery/surgery.dm

```diff
@@ -187,7 +187,7 @@
 	return null
 
 /proc/spread_germs_to_organ(obj/item/organ/E, mob/living/carbon/human/user, obj/item/tool)
-	if(!istype(user) || !istype(E) || !(E.status & ORGAN_ROBOT) || E.sterile)
+	if(!istype(user) || !istype(E) || E.is_robotic() || E.sterile)
```

Comment:
@tigercat2000 @Crazylemon64 

I suspect this was coded improperly the first time. If it's reading this correctly, `spread_germs_to_organ` will fail if the organ is ...organic.

Reply:
The fix looks correct, yes